packages/cactus-plugin-ledger-connector-aries/src/main/typescript/api-client/aries-api-client.ts

```diff
@@ -0,0 +1,427 @@
+import { Observable, ReplaySubject } from "rxjs";
+import { finalize } from "rxjs/operators";
+import { io } from "socket.io-client";
```

Comment:
@outSH When I try to run the tests this is what it crashes with:

```sh
vscode ➜ /workspaces/cacti (add_aries_connector_pr) $ yarn test:jest:all packages/cactus-plugin-ledger-connector-aries/src/test/typescript/integration/aries-setup-and-connections.test.ts
jest-haste-map: duplicate manual mock found: ConfigUtil
  The following files share their name; please delete one of them:
    * <rootDir>/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/util/__mocks__/ConfigUtil.ts
    * <rootDir>/packages/cactus-cmd-socketio-server/dist/lib/main/typescript/routing-interface/util/__mocks__/ConfigUtil.js

 FAIL  packages/cactus-plugin-ledger-connector-aries/src/test/typescript/integration/aries-setup-and-connections.test.ts
  ● Test suite failed to run

    packages/cactus-plugin-ledger-connector-aries/src/main/typescript/api-client/aries-api-client.ts:3:20 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("socket.io-client")' call instead.
      To convert this file to an ECMAScript module, change its file extension to '.mts', or add the field `"type": "module"` to '/workspaces/cacti/packages/cactus-plugin-ledger-connector-aries/package.json'.

    3 import { io } from "socket.io-client";
                         ~~~~~~~~~~~~~~~~~~

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        6.539 s
Ran all test suites matching /packages\/cactus-plugin-ledger-connector-aries\/src\/test\/typescript\/integration\/aries-setup-and-connections.test.ts/i.
vscode ➜ /workspaces/cacti (add_aries_connector_pr) $
```

I'm not a 100% sure but maybe you can fix it by importing my custom made version of the same package with the `-fixed-types` suffix in it's name? It could be that I was fixing some other issue with that package so I'm not sure but worth a try IMO

Reply:
Fixed

Reply:
@outSH Thank you!